src/main/java/com/linecorp/armeria/client/http/ImmutableHttpHeaders.java

```diff
@@ -37,7 +37,7 @@
 
     @Override
     public String get(String name) {
-        return delegate.get(name);
+        return delegate.getAsString(name);
```

Comment:
The HttpHeaders interface is currently quite messy, but this class is meant to be a simple proxy to HttpHeaders so it's better to just have it call the same method on the delegate.

Can you revert these and instead add this?
```
@SuppressWarnings("deprecation")
```